modules/visual_script/visual_script.cpp

```diff
@@ -1206,6 +1260,30 @@ void VisualScript::_set_data(const Dictionary &p_data) {
 		is_tool_script = d["is_tool_script"];
 	else
 		is_tool_script = false;
+
+	// Takes all the rpc methods
+	rpc_functions.clear();
+	rpc_variables.clear();
+	for (Map<StringName, Function>::Element *E = functions.front(); E; E = E->next()) {
+		if (E->get().function_id >= 0 && E->get().nodes.find(E->get().function_id)) {
+			Ref<VisualScriptFunction> vsf = E->get().nodes[E->get().function_id].node;
+			if (vsf.is_valid()) {
+				if (vsf->get_rpc_mode() != MultiplayerAPI::RPC_MODE_DISABLED) {
+					NetData nd;
+					nd.name = E->key();
+					nd.mode = vsf->get_rpc_mode();
+					if (rpc_functions.find(nd) == -1) {
+						rpc_functions.push_back(nd);
+					}
+				}
+			}
+		}
+	}
+
+	// Visual script doesn't have rset :(
```

Comment:
:( How do I get this in?

Reply:
oh, You have to add the class variables first because these are totally missing